packages/EasyCodingStandard/packages/FixerRunner/src/Parser/FileToTokensParser.php

```diff
@@ -6,26 +6,8 @@
 
 final class FileToTokensParser
 {
-    /**
-     * @var Tokens[]
-     */
-    private $tokensByFileHash = [];
-
     public function parseFromFilePath(string $filePath): Tokens
     {
-        $fileHash = md5_file($filePath);
-
-        if (isset($this->tokensByFileHash[$fileHash])) {
-            return $this->tokensByFileHash[$fileHash];
-        }
-
-        $content = file_get_contents($filePath);
-
-        return $this->tokensByFileHash[$fileHash] = Tokens::fromCode($content);
-    }
-
-    public function clearCache(): void
-    {
-        Tokens::clearCache();
+        return Tokens::fromCode(file_get_contents($filePath));
```

Comment:
This would be great to cache, but serialize & deserialize results in some items loss :cry: